tests/lib/init/config-initializer.js

```diff
@@ -447,4 +448,124 @@ describe("configInitializer", () => {
             });
         });
     });
+
+    describe("writeFile()", () => {
+
+        beforeEach(() => {
+            answers = {
+                purpose: "style",
+                source: "prompt",
+                extendDefault: true,
+                indent: 2,
+                quotes: "single",
+                linebreak: "unix",
+                semi: true,
+                moduleType: "esm",
+                es6Globals: true,
+                env: ["browser"],
+                format: "JSON"
+            };
+
+            pkgJSONContens = {
+                name: "config-initializer",
+                version: "1.0.0"
+            };
+        });
+
+        afterEach(() => {
+            process.chdir(originalDir);
+        });
+
+        it("should create eslintrc.json", () => {
+            process.chdir(fixtureDir);
+
+            const config = init.processAnswers(answers);
+            const filePath = path.resolve(process.cwd(), ".eslintrc.json");
```

Comment:
Can we replace all `process.cwd()` with `fixtureDir`, to prevent accidental overwriting of project files?

Reply:
Yeah sure 👍🏻